### ISSUE TITLE
Antivirus preventing access to hosts file crashes docker. 

### ISSUE THREAD

**Issue by murlakatam** [docker/for-win#2292]
- [ ] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: E7C2A52E-F463-4BA6-B1E2-350CD40A6ADE/20180727093207

### Expected behavior
docker reads host file without trying to modify it and start

### Actual behaviour
docker is crashing on start

### Information
  - Windows Version: 10 Ent 1803
  - Docker for Windows Version: latest update for windows (docker was running fine 2 updates before)

### Steps to reproduce the behavior

  1. Install Webroot secure anywhere and force hosts files protection
  2. Try to run docker

**Comment by devingoble** [comment]
I am also getting this issue on Windows 10 1803 Pro. I was on Docker 17.03, and upgraded to 18.03. I found that Docker wouldn't start. I didn't have time to troubleshoot much, so I rolled back. I ended up on 17.06, and everything worked fine. I just tried upgrading to 18.06, and I am getting the same issue.

`Access to the path 'C:\WINDOWS\System32\drivers\etc\hosts' is denied.`

The antivirus being run in is McAffee.

80B4BB6A-67F6-456B-9865-55B2ED25CCF8/20180907124336
